### PR TITLE
Update pico_observing_led.py

### PR DESCRIPTION
**Pull request by BertLindeman**
The test for scan result is missing the variable in the compare. The else will never be reached.

### CODE REVIEW

File: pico_observing_led.py [hunk 1]
```diff
@@ -9,7 +9,7 @@
 - Upload file ble_pybricks.py together with this file to your Pico.
 
 LEGO Hub setup:
-- Make shure your LEGO hub is running Pybricks.
```
Comment (KWSmit) [inline pico_observing_led.py:12]: you're correct, it's a typo...

File: pico_observing_led.py [hunk 2]
```diff
@@ -34,7 +34,7 @@
 def bt_irq(event, data):
     global receivedNumber, dataReceivedFlag, led_value
     values = []
-    if _IRQ_SCAN_RESULT:
```
Comment (KWSmit) [inline pico_observing_led.py:37]: Oops, nasty mistake. You're right again :smile:

File: pico_observing_led.py [hunk 3]
```diff
@@ -72,4 +72,4 @@ def bt_irq(event, data):
         dataReceivedFlag = False
     if led_value < 0:
         led_value = 0
-    led15.duty_u16(int(f*led_value))
```
Comment (KWSmit) [inline pico_observing_led.py:75]: I don't see any changes here, am I correct?

**Review by KWSmit — APPROVED** [review]
Thanks for helping to improve my code.

**Comment by KWSmit** [comment]
240415 Bugfix file pico_observing_led.py

Add variable `event` to first compare in function `bt_irq`.
Remove typo in comments.

**Comment by KWSmit** [comment]
240415 Bugfix file `pico_observing_led.py`

Add variable `event` to first compare in function `bt_irq`.
Remove typo in comments.

File: pico_observing_led.py (hunk 3 above)
Comment (BertLindeman) [reply pico_observing_led.py:75]: No idea what the diff would be here.
PEP8 would want an empty line at the end.
But I did not do it. (I think)
I see no change on that line.

**Pull request merged**